### PR TITLE
Website improvements

### LINKED ISSUE #4: hovering over header links leaves remains
when you hover the main header links the text decoration leaves some remains
image

### PR DESCRIPTION
**Pull request by AJCM-git**
Adds:
- [x] Table of contents

- [x] Image and video styling 

- [x] Moves the important info to the header (discord, github and other importants links)

- [x] Media section where all the images and videos will be, so that they are easily accesible by users

- [x] Change footer text links with images

- [x] Align correctly the downloads

- [x] Contact page

- [x] Put a title to the videos

- [x] Select better media for the media page

- [x] Add a email, IRC, matrix and subreddit links to the footer

- [x] Fix #4 

~~- [x] monero miner~~
P.D: I added a `max-width` and `min-width` to images and videos so that the outline didnt go outside the page, this may affect the responsive-side of the website? is there a better solution?
P.D2: I couldnt get the images to center

Known issues:

~~- When zooming out and then in, the images of the header links are put above the text~~
~~- Media toc link is hardcoded~~

### CODE REVIEW

File: assets/css/main.css [hunk 1]
```diff
@@ -277,6 +277,383 @@ code {
 	--number: #b5cea8;
 }
 
+body
```
Comment (PJB3005) [inline assets/css/main.css:280]: You just duplicated in the entire (old) stylesheet. Please fix this.

Comment (AJCM-git) [reply assets/css/main.css:280]: Yeah, i just realized, i fucked up the git config and had to move the things to a new repository

**Comment by AJCM-git** [comment]
This is ready for ~~instamerge~~ a review!

**Comment by PJB3005** [comment]
Fix the conflicts first, please.

**Comment by AJCM-git** [comment]
Conflicts gone
image

**Comment by AJCM-git** [comment]
The header should be centered now, i dumped the border and put discord link at the top of the contacts page. I re-added the thumbnails, i dumped them before because they lagged my browser for like 2 seconds the first time visiting the page.

**Pull request merged**